wafer/talks/templates/wafer.talks/talk.html

```diff
@@ -41,6 +41,13 @@
 <div class="well">
     {{ object.abstract.rendered|safe }}
 </div>
+{% if show_private_notes and perms.edit_private_notes and object.private_notes %}
```

Comment:
Couple of thoughts on this:
* I'm still not wild about showing the private notes here. Anyone want to tie-break for madduck and I?
* I think the `<h2>` heading needs to be inside the `<div class="well">` for consistency with what happens with other headings generated by the talk abstract itself?